Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -305,6 +322,36 @@ try {
                     $srcContent = Get-ContentLF -Path $srcFile
                 }
 
+                $srcContent = $srcContent.Replace('{TEMPLATEURL}', "$($templateUrl)@$($templateBranch)")
+                if ($directALGo) {
+                    # If we are using the direct AL-Go repo, we need to change the owner and repo names in the workflow
+                    $lines = $srcContent.Split("`n")
+                    
+                    # The Original Owner and Repo in the AL-Go repository are microsoft/AL-Go-Actions, microsoft/AL-Go-PTE and microsoft/AL-Go-AppSource
+                    $originalOwnerAndRepo = @{
+                        "actionsRepo" = "microsoft/AL-Go-Actions"
+                        "perTenantExtensionRepo" = "microsoft/AL-Go-PTE"
+                        "appSourceAppRepo" = "microsoft/AL-Go-AppSource"
+                    }
+                    # Original branch is always main
+                    $originalBranch = "main"
```

Comment:
Is this safe to assume? What if we are working on a fix for specific version?

Reply:
The pointers in microsoft/AL-Go/Templates/Per Tenant Extensions is always main - and should always be main

Reply:
Okay, I'll trust you on this one ;)

Reply:
You don't need to trust me - look in a forked, branched version of AL-GO:
This is a normal fork after deploying and collecting:
https://github.com/freddydk/AL-Go/blob/jobname/Templates/Per%20Tenant%20Extension/.github/workflows/CICD.yaml

This is the devprocess fork I used for testing this in BusinessCentral/BingMaps.PTE:
https://github.com/freddydk/AL-Go/blob/devprocess/Templates/Per%20Tenant%20Extension/.github/workflows/CICD.yaml

Always main. The change is only done when deploying and collecting - and in this PR, the change is also done when updating AL-Go System Files with an AL-Go repo instead of a template repo.

AL-Go repo needs to be main in order to be able to create a PR on microsoft/AL-Go - no changes ever.